TYPO3.Neos/Classes/TYPO3/Neos/ViewHelpers/Link/NodeViewHelper.php

```diff
@@ -129,7 +129,7 @@ public function initializeArguments()
     /**
      * Renders the link. Renders the linked node's label if there's no child content.
      *
-     * @param mixed $node A node object or a string node path or NULL to resolve the current document node
+     * @param mixed $node A node object or a string node path(absolute or relative) a string node://-uri or NULL to resolve the current document node
```

Comment:
ADDITIONALLY: The Doc Comment up there in the class must be updated; as this is used to generate the RTD Docs.